### ISSUE TITLE
Support RunBlocking (Ktor Network calls in a Synchronous manner)

### ISSUE THREAD

**Issue by cody1024d** [ktorio/ktor#1421]
I was hoping things have potentially changed since this: https://github.com/ktorio/ktor/issues/678

I tried using the work around described there, but to no success.

My use case is as follows:

I am writing a framework to be used during the testing process of IOS and Android.  Within this process I will need to perform several network calls (for authentication reasons) before my tests can execute, and thus I need to run Ktor in a blocking manner.

Is there any work arounds, etc. that can be done to support this?

**Comment by uOOOO** [comment]
Sometimes only network call methods should be exposed not whole MV* pattern in shared code. So I think this is very necessary feature.

**Comment by bootstraponline** [comment]
I agree this would be awesome. We're thinking about Kotlin multiplatform at Robinhood. This is one of the blocking issues for us.

**Comment by oleg-larshin** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/KTOR-534">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
